splice_protocol/src/main/protobuf/Splice.proto

```diff
@@ -160,6 +160,19 @@ message SpliceRegionSizeResponse {
     required int64 sizeInBytes = 2;
 }
 
+service SpliceRegionServerCoprocessorService {
+	rpc getOldestActiveTransaction(SpliceOldestActiveTransactionRequest)
+		returns (SpliceOldestActiveTransactionResponse);
+}
+
+message SpliceOldestActiveTransactionRequest {
+
+}
+
+message SpliceOldestActiveTransactionResponse {
+    required int64 oldestActiveTransaction = 1;
```

Comment:
In order to garbage collect data, you will probably need the most recent transaction for each server when there's no active transactions. I'm ok adding that later though.